src/js/disability-benefits/components/ClaimsListItem.jsx

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 import { Link } from 'react-router';
+import moment from 'moment';
+
+import { getPhaseDescription } from '../utils/helpers';
 
 export default function ClaimsListItem({ claim }) {
-  const filesNeeded = claim.attributes.trackedItems.stillNeedFromYouList ? claim.attributes.trackedItems.stillNeedFromYouList.length : null;
   return (
     <Link className="claim-list-item" to={`/your-claims/${claim.id}/status`}>
       <h4 className="claim-list-item-header">Compensation Claim</h4>
-      <p className="status">Status: {claim.attributes.phaseDates.latestPhaseType}</p>
-      {filesNeeded !== null
-        ? <p><i className="fa fa-exclamation-triangle"></i>We need {filesNeeded} {filesNeeded > 1 ? 'files' : 'file'} from you</p>
-        : null}
-      <p><i className="fa fa-envelope"></i>We sent you a development letter (TODO)</p>
-      <p>Last Update: TODO</p>
+      <p className="status">Status: {getPhaseDescription(claim.attributes.phase)}</p>
+      <div className="communications">
+        {claim.attributes.developmentLetterSent && !claim.attributes.decisionLetterSent
+          ? <p><i className="fa fa-envelope"></i>We sent you a development letter</p>
+          : null}
+        {claim.attributes.decisionLetterSent
+          ? <p><i className="fa fa-envelope"></i>We sent you a decision letter</p>
+          : null}
+        {!claim.attributes.decisionLetterSent && claim.attributes.documentsNeeded
+          ? <p><i className="fa fa-exclamation-triangle"></i>We need files from you</p>
+          : null}
+      </div>
+      <p>Last update: {moment(claim.attributes.updatedAt).format('MMM M, YYYY')}</p>
```

Comment:
I'm not sure which date we want to show here, the last time we updated data from EVSS or the last update to the claim. If it's the last time we synced data from EVSS, this is correct. If it's the last update to the claim, I think you'd use the most recent date from the `events_list`